### PR TITLE
handle unknown status

### PR DESCRIPTION
**Pull request by ldpliu**
https://issues.redhat.com/browse/ACM-5729
https://issues.redhat.com/browse/ACM-5730

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-cluster-group-compliancy-overview.yaml [hunk 1]
```diff
@@ -125,7 +125,7 @@ data:
...
+              "rawSql": "WITH data as (\n  SELECT\n    ch.compliance_date as \"time\",\n    payload -> 'metadata' -> 'labels' ->> '$label' AS \"$label\",\n    COUNT(CASE WHEN ch.compliance = 'non_compliant' THEN 1 END) AS \"non_compliant\",\n    COUNT(CASE WHEN ch.compliance = 'unknown' THEN 1 END) AS \"unknown\",\n    COUNT(CASE WHEN ch.compliance = 'compliant' THEN 1 END) AS \"compliant\"\n  FROM\n    status.managed_clusters mc\n  JOIN\n    local_status.compliance_history ch ON mc.cluster_id = ch.cluster_id\n  WHERE\n    payload -> 'metadata' -> 'labels' ->> '$label' IN ($value)\n  GROUP BY (ch.compliance_date, mc.payload -> 'metadata' -> 'labels' ->> '$label')\n  ORDER BY (ch.compliance_date)\n)\nSELECT\n  time,\n  $label as \"metric\",\n  compliant::float / (unknown::float + compliant::float + non_compliant) as \"value\"\nFROM\n  data\nORDER BY (time)",
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-cluster-group-compliancy-overview.yaml:128]: We have merged a PR yesterday to move table `local_status.compliance_history` to `history.local_compliance`.

Comment (ldpliu) [reply operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-cluster-group-compliancy-overview.yaml]: Changed all places where used this table.

File: operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-offending-clusters.yaml [hunk 2]
```diff
@@ -200,6 +192,86 @@ data:
           "title": "Number of Compliant Managed Clusters",
           "type": "stat"
         },
+        {
+          "datasource": null,
+          "description": "Total number of managed clusters that were  unknown.",
```
Comment (clyang82) [inline operator/pkg/controllers/hubofhubs/manifests/grafana/acm-global-offending-clusters.yaml:197]: remove the redundant space.

**Review by clyang82 — CHANGES_REQUESTED** [review]

**Comment by clyang82** [comment]
BTW, could you append a picture about the changed dashboard?

**Comment by hchenxa** [comment]
+1 with @clyang82 , a snapshot will be good to let us know the new changes.

**Comment by ldpliu** [comment]
fc
gp

**Comment by ldpliu** [comment]
@clyang82 @hchenxa I have uploaded the snapshot, please help to review it.

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**